_dashboards/selfhost-maps-server.md

```diff
@@ -0,0 +1,98 @@
+
+You must set up the map tiles before running the server. You have two setup options: (1) use the OpenSearch-provided maps service tiles set or (2) generate the raster tile set yourself.
+
+### Option 1: Use the OpenSearch-provided maps service tiles set
+
+Create a Docker volume to hold the tiles set:
+
+`docker volume create tiles-data`
+
+Download the tiles set from the OpenSearch maps service (planet tiles sets [0-8](https://maps.opensearch.org/offline/planet-osm-default-z0-z8.tar.gz) and [0-10](https://maps.opensearch.org/offline/planet-osm-default-z0-z10.tar.gz) are available):
+
+```
+docker run \
+-e DOWNLOAD_TILES=https://maps.opensearch.org/offline/planet-osm-default-z0-z8.tar.gz \
+-v tiles-data:/usr/src/app/public/tiles/data/ \
+opensearch/opensearch-maps-server \
+import
+```
+
+### Option 2: Generate the raster tile set
+
+To generate the raster tiles images set, use the [raster tile generation pipeline](https://github.com/opensearch-project/maps/tree/main/tiles-generation/cdk) and then use the tile set absolute path to create a volume to start the server.
+
+## Start the server
+
+Use the following command to start the server using the Docker volume `tiles-data`. The command below is an example using host url "localhost" and port "8080."  
+
+```
+docker run \
+-v tiles-data:/usr/src/app/public/tiles/data/ \
+-e HOST_URL='http://localhost' \
+-p 8080:8080 \
+opensearch/opensearch-maps-server \
+run
+```
+
+Or, if you generated raster tiles images, run the server using that tiles set:
+
+```
+docker run \
+-v /absolute/path/to/tiles/:/usr/src/app/dist/public/tiles/data/ \
+-p 8080:8080 \
+opensearch/opensearch-maps-server \
+run
+```
+
+To confirm the server is running, you should be able to access the following files on your localhost:  
+
+* [Maps manifest](http://localhost:8080/manifest.json)
+* [Maps tiles](http://localhost:8080/tiles/data/%7Bz%7D/%7Bx%7D/%7By%7D.png)
+* [Maps tiles demo](http://localhost:8080/) 
+
+## Use the self-host maps server with OpenSearch Dashboards
+
+You can use the self-host maps server with OpenSearch Dashboards by either adding the parameter to `opensearch_dashboards.yml` or configuring the default WMS properties in OpenSearch Dashboards.
+
+### Option 1: Configure opensearch_dashboards.yml
+
+Configure the manifest url in `opensearch_dashboards.yml`:
+
+`map.opensearchManifestServiceUrl: "http://localhost:8080/manifest.json"`
+
+### Option 2: Configure Default WMS properties in OpenSearch Dashboards
+
+1. On the OpenSearch Dashboards console, select **Stack Management > Advanced Settings**. 
+2. Locate `visualization:tileMap:WMSdefaults` under **Default WMS properties**. 
+3. Change enabled to `true` and add the URL for the valid map server.
+
+## Licenses
+
+Tiles are generated per [Terms of Use for Natural Earth raster and vector map data](https://www.naturalearthdata.com/about/terms-of-use/) and [Copyright and License for OpenStreetMap](https://www.openstreetmap.org/copyright).
```

Comment:
From Natural Earth, We only vector map data from them. So we could say:

Tiles are generated per [Terms of Use for Natural Earth](https://www.naturalearthdata.com/about/terms-of-use/) vector map data and [Copyright and License for OpenStreetMap](https://www.openstreetmap.org/copyright). 

Thanks!